packages/rich-text-types/src/__test__/validation.test.ts

```diff
@@ -24,6 +24,39 @@ const text = (value = '', args?: any) => ({
   ...args,
 });
 
+const topLevelBlocks = [
```

Comment:
There's actually constants for both of these exported by this package `TOP_LEVEL_BLOCKS` and `LIST_ITEM_BLOCKS` :laughing: